source/_integrations/nina.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: NINA
+description: Instructions on how to set up NINA warnings in Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_release: 2021.11.0
```

Comment:
```suggestion
ha_release: 2022.01.0
```

Reply:
This suggestion is incorrect. We don't prefix the minor part of releases with a leading `0`, additional, there will not be a `2022.1` release, as the first upcoming release will be `2022.2`.

Reply:
Lets just use your suggestion then :+1: